src/js/components/BreadcrumbComponent.jsx

```diff
@@ -12,13 +14,78 @@ var BreadcrumbComponent = React.createClass({
     taskId: React.PropTypes.string
   },
 
+  getInitialState: function () {
+    return {
+      collapsed: false
+    };
+  },
+
+  componentDidMount: function () {
+    // Avoid referencing window from Node context
+    if (global.window != null) {
+      window.addEventListener("resize", this.handleResize);
```

Comment:
This should work. :)
To avoid some misbehaviors it's also necessary to listen on the "focus"-event additionally.
```javascript
window.addEventListener("focus", this.handleResize);
```